eZ/Publish/Core/Persistence/Cache/ContentHandler.php

```diff
@@ -317,4 +317,27 @@ public function removeTranslationFromContent($contentId, $languageCode)
         $this->cache->clear('content', $contentId);
         $this->cache->clear('content', 'info', $contentId);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function deleteTranslationFromVersion($contentId, $versionNo, $languageCode)
+    {
+        $this->logger->logCall(
+            __METHOD__,
+            ['content' => $contentId, 'version' => $versionNo, 'languageCode' => $languageCode]
+        );
+        $content = $this->persistenceHandler->contentHandler()->deleteTranslationFromVersion(
+            $contentId,
+            $versionNo,
+            $languageCode
+        );
+        $this->cache->clear('content', $contentId, $versionNo);
+        $this->cache
+            ->getItem('content', $contentId, $versionNo, self::ALL_TRANSLATIONS_KEY)
+            ->set($content)
```

Comment:
1. Actually, could you skip the cache warming?
2. versionNo is wrong here and above, cause the returned item is a new version right? and as it's not published afaik we don't need to clear any cache _(same case as `createDraftFromVersion`)_

Reply:
> Also versionNo is wrong here and above right? Cause the returned content is a new draft version, and as it's not published so afaik we don't need to clear any cache (same case as createDraftFromVersion)

Version passed to this method is a draft, so `versionNo` is correct. It works in a similar way like `updateContent` from which I copied cache clearing and warming. 

However I don't get cached value while "playing" with REST requests on draft (after `deleteTranslation` and before publishing), so maybe it's really not needed there.

Reply:
On point 2: My bad, I was miss reading the concepts and thinking the method here creates a new version for you w/o the given translation.

So just skip the cache warmin then, meaning only call needed is:
`this->cache->clear('content', $contentId, $versionNo);`


As for not getting cached value, unsure what you are actually testing, but if what you are testing triggers `load()` on this handler, it is cached. If it just uses ´ loadVersionInfo´ then it's not in 1.x (it is in 2.x).

Reply:
Ok, fixed: https://github.com/ezsystems/ezpublish-kernel/pull/2081/commits/561938de942cf23c2549cd31e2ebd257194b452e#diff-fb693bf163e578a01367f4466a44ad9b